setup.py

```diff
@@ -56,7 +56,7 @@
 dev_requirements = explode('dev_tools/requirements/deps/dev-tools.txt')
 
 # filter out direct urls (https://github.com/pypa/pip/issues/6301)
-dev_requirements = [r.strip() for r in dev_requirements if "git+http" not in r]
+dev_requirements = [r.strip() for r in dev_requirements if "http" not in r]
```

Comment:
Might be better to do something like `https://` to be sure this catches urls only, since a standard pypi package could include the string "http".

Reply:
Done.